lib/html/static/js/search.js

```diff
@@ -101,7 +100,7 @@ var FileLoader = ( function () {
     this.fileSearch = function () {
         /* Called on submit by form:file_search to initiate a search. */
         var search_string = document.forms['file-search']['search-string'].value;
-        var search_mode = document.forms['file-search']['search-mode'].value;
+        var search_mode = $(document.forms['file-search']['search-mode']).val();
```

Comment:
Its because of the way search_mode is changed (via the dropdown in the search box), for some reason this doesn't work the pure js way.